src/ensenso_driver.cpp

```diff
@@ -43,13 +45,15 @@ class EnsensoDriver
     image_transport::CameraPublisher  r_raw_pub_;
     image_transport::Publisher        l_rectified_pub_;
     image_transport::Publisher        r_rectified_pub_;
+    ros::Publisher                    disparity_pub_;
```

Comment:
It's a disparity image, so is it better grouped with other images, or by the transport mechanism?